ecodev_core/list_utils.py

```diff
@@ -45,6 +45,45 @@ def first_or_default(sequence: Union[List[Any], None],
     return next((elt for elt in sequence if condition(elt)), default)
 
 
+def sort_by_keys(unsorted_dict: dict, reverse: bool = False) -> dict:
```

Comment:
Curious as to the use case here as I don't think I've ever seen a need for a sorted dictionary?

Reply:
From keys it's rarer, but it happens from time to time :)

Reply:
I have been 54 times on https://stackoverflow.com/questions/9001509/how-do-i-sort-a-dictionary-by-key ahahah

Reply:
it's a very popular one it seems! I dunno what rock I've been hiding under hahaha